backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/components/NodeComponent.java

```diff
@@ -78,6 +82,17 @@ public Element render() {
         return new Fragment(fragmentProps);
     }
 
+    private boolean viewCreationRequestedFor(Object semanticElement) {
+        NodeDescription nodeDescription = this.props.getNodeDescription();
+        VariableManager variableManager = this.props.getVariableManager();
+        Optional<Diagram> optionalDiagramContainer = variableManager.get("ContainerView", Diagram.class); //$NON-NLS-1$
+        Optional<Node> optionalNodeContainer = variableManager.get("ContainerView", Node.class); //$NON-NLS-1$
```

Comment:
`"ContainerView"` should be extracted in a constant.

Reply:
This code has been removed after the SBE comment https://github.com/eclipse-sirius/sirius-components/pull/208#discussion_r546930753